httpd/handler.go

```diff
@@ -248,7 +248,7 @@ func (h *Handler) serveStatus(w http.ResponseWriter, r *http.Request) {
 	} else {
 		b, _ = json.Marshal(data)
 	}
-	w.Write(b)
+	_, _ = w.Write(b)
```

Comment:
I don't think we can ignore errors based on name. The two times when I ignore returned errors are when I'm dealing with something like `bytes.Buffer` which cannot return an error and when I'm writing out to an `http.ResponseWriter` where there's not really much to be done if it's not successful.